### DISCUSSION TITLE
Welcome to settings Discussions!

### DISCUSSION THREAD

**Discussion by travi** [repository-settings/app#450] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Crazypantz** [comment]
Autumn G and I'm still fumbling my way around.

**Comment by travi** [comment]
Hi everyone 👋🏼 

I'm the primary maintainer of the settings app. I am not the original creator, but have been a heavy user for long enough that I am invested in it continuing to exist, so I [volunteered to help maintain](https://github.com/probot/probot/issues/1022). Since I am a volunteer, my time availability is admittedly a bit limited, so please be patient with me.

Outside of this project, I am quite focused on [DivOps](https://www.jonathancreamer.com/announcing-div-ops/#what-am-i-even). I spend a fair amount of effort [codifying maintenance details for my projects](https://github.com/form8ion), automating dependency updates, and releasing new versions of packages with [semantic-release](https://github.com/semantic-release), where I am also a core maintainer. 

I'm hoping to take a few steps to help make this project a bit more sustainable soon, so keep an eye out for some of those changes.

**Comment by johnmartel** [comment]
Hi @travi and everyone 👋 

I am really interested in this repository and follow closely its evolution as I am a huge advocate of as-much-as-possible-as-code at my current job. Most people I work with have adopted the settings app too.

I am not a maintainer and did not actually contribute a PR yet, but I like to chip in when I can spare the time and help others with stuff I have figured out after quite a few months using this app on all repositories I have been involved with.
